index.tt

```diff
@@ -13,7 +13,7 @@
     <p class="lead">
       NixOS, an open source software, gives you tools to share
       your development and build environments between machines with confidence.
-      NixOS GNU/Linux distribution is built for the cloud, that brings
+      NixOS GNU/Linux distribution is built for the cloud and brings
```

Comment:
words words words

is "cloud" really something we want to use?

like "it's just someone elses computer". or do we need some buzzwords to please some people?

Reply:
Agree cloud is weird word, but couldn't come up with something better. We need to change this part.